jenkins-pipelines/longevity-50gb-asymmetric-cluster-12h.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-50GB-12h-verifier-LimitedMonkey-tls.yaml", "configurations/db-nodes-shards-random.yaml"]''',
+
+    timeout: [time: 780, unit: 'MINUTES']
```

Comment:
remove

Reply:
we use test duration to dynamically calculate the sct timeout

Reply:
done